src/BlazingPizza.Client/Pages/Checkout.razor

```diff
@@ -10,15 +10,25 @@
             <h2>Signing in...</h2>
         </NotAuthorized>
         <Authorized>
-            <h2>Checkout</h2>
+            <EditForm Model="@OrderState.Order.DeliveryAddress" OnValidSubmit="@SubmitOrder" Context="form">
```

Comment:
`Context="form"` - think this is unused.

Reply:
It’s required otherwise the compiler errors, because you have overlapping contexts both called “context”. 

We should probably relax the name clash rule in the case of the name “context”.